### PR TITLE
Migrate from Travis CI to GitHub Actions

### PR DESCRIPTION
**Pull request by mymindstorm**
Quick math shows that one of our builds on travis takes about 4 min, meaning that with new travis pricing we can only run 250 builds a month. I want to eventually have integration testing on CI which will eat more minutes, and GH actions should help with that.

### CODE REVIEW

**Comment by Sneezry** [comment]
Shall we keep travis while migrating to github actions? And remove travis after github actions work stable.

**Comment by mymindstorm** [comment]
@Sneezry the parts that haven't been tested cannot run in parallel (i18n strings auto-commit, tagging, and auto-release).

**Comment by mymindstorm** [comment]
In case this doesn't work at all we can always just revert to travis.

**Comment by Sneezry** [comment]
Is this PR ready to review?

**Comment by mymindstorm** [comment]
Yes

File: .github/workflows/i18n.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: i18n
+
+on:
+  push:
+    branches: [ dev ]
+
+jobs:
+  i18n-strings:
+    runs-on: ubuntu-latest
+    name: Process new i18n strings`
```
Comment (Sneezry) [inline .github/workflows/i18n.yml]: extra ` in end of line

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      
+      - name: Install dependencies
+        run: |
+          npm i prettier
+          sudo npm i -g lintspaces-cli
+
+      - name: Prettier
+        run: ./node_modules/prettier/bin-prettier.js --check ./src/* ./src/**/* ./src/**/**/* ./sass/*.scss
+
+      - name: lintspaces
```
Comment (Sneezry) [inline .github/workflows/main.yml:28]: Upper case first letter

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        run: |
+          lintspaces -nt -d 'spaces' -i 'js-comments' src/*/* src/*.ts view/* manifest-*.json css/popup.css css/import.css
+  build-ffx:
+    runs-on: ubuntu-latest
```
Comment (Sneezry) [inline .github/workflows/main.yml:32]: Do we need set up new environment for build?

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      
+      - name: Install dependencies
+        run: npm i
+      
+      - name: Build
+        run: bash scripts/release.sh
+        env:
+          CREDS_FILE_PASSWORD: ${{ secrets.CREDS_FILE_PASSWORD }}
+      
+      - name: Create a Release
```
Comment (Sneezry) [inline .github/workflows/release.yml]: Create a release

**Review by Sneezry — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 2 above)
Comment (mymindstorm) [reply .github/workflows/main.yml:28]: This is intentional

File: .github/workflows/main.yml (hunk 3 above)
Comment (mymindstorm) [reply .github/workflows/main.yml:32]: No, each key under jobs is run in a separate environment.

**Review by Sneezry — APPROVED** [review]

**Pull request merged**